pkg/slayers/scion_spec.gobra

```diff
@@ -349,6 +349,29 @@ func (s *SCION) GetPath(ub []byte) path.Path {
 	return unfolding acc(s.Mem(ub), _) in s.Path
 }
 
+ghost
+opaque
+pure
+requires acc(s.Mem(ub), _)
+requires acc(slices.AbsSlice_Bytes(ub, 0, len(ub)), _)
+decreases
+func (s *SCION) EQAbsHeader(ub []byte) bool {
```

Comment:
```suggestion
func (s *SCION) EqAbsHeader(ub []byte) bool {
```

Feel free to ignore this suggestion and all others for this name for now. we can do renamings in a future pr